prody/proteins/interactions.py

```diff
@@ -0,0 +1,2043 @@
+    """Compute Pi-cation interactions for DCD trajectory using default paramaters.
+        
+    :arg atoms: an Atomic object from which residues are selected
+    :type atoms: :class:`.Atomic`
+        
+    :arg trajectory: DCD filename
+    :type trajectory: str"""
+
+    try:
+        coords = (atoms._getCoords() if hasattr(atoms, '_getCoords') else
+                    atoms.getCoords())
+    except AttributeError:
+        try:
+            checkCoords(coords)
+        except TypeError:
+            raise TypeError('coords must be an object '
+                            'with `getCoords` method')
+        
+    if not isinstance(trajectory, (TrajBase, Ensemble, Atomic)):
+        raise TypeError('{0} is not a valid type for trajectory'
+                    .format(type(trajectory)))
+                        
+    pi_cat_all = []
+        
+    for j0, frame0 in enumerate(trajectory):  
+        LOGGER.info('Frame: {0}'.format(j0))
+        protein = atoms.select('protein')
+        pi_cat = calcPiCation(protein, distA, extraSele, **kwargs)
+        pi_cat_all.append(pi_cat)
+        
+    return pi_cat_all
+
+
+def calcHydrophohicDCD(atoms, trajectory, distA=4.5, **kwargs):  
+    """Compute hydrophobic interactions for DCD trajectory using default paramaters.
+        
+    :arg atoms: an Atomic object from which residues are selected
+    :type atoms: :class:`.Atomic`
+        
+    :arg trajectory: DCD filename
+    :type trajectory: str"""
+
+    try:
+        coords = (atoms._getCoords() if hasattr(atoms, '_getCoords') else
+                    atoms.getCoords())
+    except AttributeError:
+        try:
+            checkCoords(coords)
+        except TypeError:
+            raise TypeError('coords must be an object '
+                            'with `getCoords` method')
+        
+    if not isinstance(trajectory, (TrajBase, Ensemble, Atomic)):
+        raise TypeError('{0} is not a valid type for trajectory'
+                    .format(type(trajectory)))
+                        
+    HPh_all = []
+        
+    for j0, frame0 in enumerate(trajectory):  
+        LOGGER.info('Frame: {0}'.format(j0))
+        protein = atoms.select('protein')
+        HPh = calcHydrophohic(protein, distA, **kwargs)
+        HPh_all.append(HPh)
+        
+    return HPh_all
+
+
+def compareInteractions(data1, data2, **kwargs):
+    """Comparison of two outputs from interactions. 
+    It will provide information about the disappearance and appearance of some interactions
+    as well as the similarities in the interactions
+    for the same system. Two conformations can be compared.
+    
+    :arg data1: list with interactions from calcHydrogenBonds() or other types
+    :type data1: list
+ 
+    :arg data2: list with interactions from calcHydrogenBonds() or other types
+    :type data2: list 
+    
+    Example of usage: 
+    >>> atoms1 = parsePDB('PDBfile1.pdb').select('protein')
```

Comment:
sometimes there are triple > like here and sometimes double ones. VS code highlights triple ones in purple so maybe they should all be like that. Perhaps check rst guidelines and/or other modules